README.rst

```diff
@@ -77,19 +84,19 @@ For inflection class lattices, both can be kept: ::
     /$ qumin pats.defective=True pats.overabundant=True data=<dataset.package.json>
 
 Microclasses
-^^^^^^^^^^^^^
+^^^^^^^^^^^^
 
 To visualize the microclasses and their similarities, one can compute a **microclass heatmap**::
 
     /$ qumin action=heatmap data=<dataset.package.json>
 
-This will compute patterns, then the heatmap. To pass pre-computed patterns, pass the file path: ::
+This will compute patterns, then the heatmap. To pass pre-computed patterns, pass the patterns computation metadata path: ::
 
-    /$ qumin action=heatmap patterns=<path/to/patterns.csv> data=<dataset.package.json>
+    /$ qumin action=heatmap patterns=<path/to/metadata.json> data=<dataset.package.json>
 
 It is also possible to pass class labels to facilitate comparisons with another classification: ::
 
-    /$ qumin.heatmap label=inflection_class patterns=<path/to/patterns.csv> data=<dataset.package.json>
+    /$ qumin.heatmap label=inflection_class patterns=<path/to/metadata.json> data=<dataset.package.json>
 
 The label key is the name of the column in the Paralex `lexemes` table to use as labels.
 
```

Comment:
Add a warning here to signal that the change to path/to/metadata.json from /path/to/patterns.csv is a breaking change with Qumin V2